lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -226,19 +227,30 @@
 .forum-nav-thread-link {
   display: block;
   padding: ($baseline/4) ($baseline/2);
+  transition: none;
 
-  &.is-active {
-    background-color: $forum-color-background;
-    color: $base-font-color;
+  &.is-active,
+  &:hover {
+    color: $forum-color-background;
+    background-color: $forum-color-hover-thread;
```

Comment:
Can you have a separate ``$forum-color-active-thread`` for themes that might want them to be different. IMO it reads strangely to say that the active thread is always the hover color.